experiment_no_Keithley_1000_Oe_field_scan.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb 28 14:53:09 2024
+
+@author: Marcus Edwards, Biswas
+"""
+import MultiPyVu as mpv
+import time
+import numpy as np
+from enum import Enum, auto
+
+
+class MVUInstrumentList(Enum):
+    DYNACOOL = auto()
+    PPMS = auto()
+    PPMSMVU = auto()
+    VERSALAB = auto()
+    MPMS3 = auto()
+    OPTICOOL = auto()
+    na = auto()
+
+mpv.instrument.InstrumentList = MVUInstrumentList
+
+def save_temp_field_chamber():
```

Comment:
We also use this function a lot. Let's import it instead of repeating this block of code.